ansible/roles/ocp-workload-fsi-cc-dispute-demo/tasks/workload.yml

```diff
@@ -4,13 +4,19 @@
     ocp_project: "fsi-cc-dispute-{{guid}}"
 
     # Templates come from here: https://raw.githubusercontent.com/jorgemoralespou/ose-sample-apps-layouts
+- name: Check if project exists
+  shell: |
+         oc get project {{ocp_project}}
+  changed_when: false
+  register: project_exists_result
+  ignore_errors: true
```

Comment:
here i would use `failed_when: false` instead of `ignore_errors` so it does not display as error when user run the playbook